src/components/structures/MatrixChat.js

```diff
@@ -1052,8 +1055,7 @@ module.exports = React.createClass({
                     page_element = (
                         <RoomView
                             ref="roomView"
-                            roomId={this.state.currentRoom}
-                            roomAlias={this.state.currentRoomAlias}
+                            roomAlias={this.state.currentRoom || this.state.currentRoomAlias}
```

Comment:
Yup; see also the PR description.

As for whether `currentRoomAlias` is ever present without `currentRoom`: ¯\_(ツ)_/¯. The logic in `MatrixChat` is opaque to me, and I was rather hoping to avoid poking at it with some minimal changes; hence this defensiveness.